src/sonic-config-engine/portconfig.py

```diff
@@ -246,53 +246,13 @@ def parse_platform_json_file(hwsku_json_file, port_config_file, interface_name=N
     for intf in port_dict[INTF_KEY]:
         if intf not in hwsku_dict[INTF_KEY]:
             raise Exception("{} is not available in hwsku_dict".format(intf))
-        if str(interface_name) == intf:
-            brkout_mode = target_brkout_mode
-        else:
-            brkout_mode = hwsku_dict[INTF_KEY][intf][BRKOUT_MODE]
-
-        index = port_dict[INTF_KEY][intf]['index']
-        alias_at_lanes = port_dict[INTF_KEY][intf]['alias_at_lanes']
-        lanes = port_dict[INTF_KEY][intf]['lanes']
 
-        # if User does not specify brkout_mode, take default_brkout_mode from hwsku.json
-        if brkout_mode is None:
-            brkout_mode = hwsku_dict[INTF_KEY][intf][BRKOUT_MODE]
+        # take default_brkout_mode from hwsku.json
+        brkout_mode = hwsku_dict[INTF_KEY][intf][BRKOUT_MODE]
 
-        # Get match_list for Asymmetric breakout mode
-        if re.search("\+",brkout_mode) is not None:
-            brkout_parts = brkout_mode.split("+")
-            match_list = [re.match(BRKOUT_PATTERN, i).groups() for i in brkout_parts]
+        child_ports = get_child_ports(intf, brkout_mode, port_config_file)
+        ports.update(child_ports)
 
-        # Get match_list for Symmetric breakout mode
-        else:
-            match_list = [re.match(BRKOUT_PATTERN, brkout_mode).groups()]
-
-        """
-        Example of match_list for some breakout_mode using regex
```

Comment:
Move this comment section to get_child_ports() method?

Reply:
done